test/dual_azure.cs

```diff
@@ -87,9 +86,17 @@ public async Task small_file_azure()
 
             var prefix = NewPersistContainer.Uri + "/a/B2EA9F7FCEA831A4A63B213F41A8855B";
             Assert.Equal(prefix, url.Substring(0, prefix.Length));
-
-            var suffix = "&sp=r&rsct=application%2Foctet-stream&rscd=attachment%3Bfilename%3D\"test.bin\"";
-            Assert.Equal(suffix, url.Substring(url.Length - suffix.Length));
+            
+            // HINT: suffix is no longer constructed in the same order for the download URL.
```

Comment:
(same comment as above)

I understand that the suffix is no longer constructed in the same order, however if the construction is deterministic, we would prefer an equality assertion instead of multiple "contains" assertions: 

 - Equality detects any changes in the suffix (including the appearance of extraneous characters)
 - The expected shape of the suffix is much more obvious with equality than with contains